### ISSUE TITLE
detach everything before render to avoid event missing.

### ISSUE THREAD

**Issue by anianj** [marionettejs/backbone.marionette#2789]
the default attachElContent of Item-View is:
 
attachElContent: function(html) {
    this.$el.html(html);
    return this;
}
This will cause all your events bind to the subviews to miss.

make it as the following will avoid the missing of events on subviews:

attachElContent: function(html) {
    this.$el.conents().detach()
    this.$el.html(html);
    return this;
}

Should me make a pull request for this?

**Comment by paulfalgout** [comment]
In what cases do you have subviews within the content of an itemview?
FWIW `detach` is what a region is doing when you replace the el.

**Comment by paulfalgout** [comment]
So I think in the typical case the region is handling this as you'd expect, and then you could override `attachElContent` for a special use case, but let me know if I'm missing something.
